app/views/common/os_selection/_architecture.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= select_f f, :operatingsystem_id, arch_oss, :id, :to_label,
     {:selected => item.operatingsystem_id, :include_blank => blank_or_inherit_f(f, :operatingsystem)},
     { :label => _("Operating system"),
-      :disabled => arch_oss.empty? ? true : false,
+      :readonly => arch_oss.empty? ? true : false,
```

Comment:
Isn't ` :readonly => arch_oss.empty?,` is enough?

Reply:
It is, but since this is a select, the readonly attribute has no effect at all. I suggest completely removing this attribute, since having no options to select should be enough, I don't think we really have to disable the dropdown completely.